docs/molnet_pr_template.md

```diff
@@ -0,0 +1,12 @@
+### Template for pull request contributing a new dataset to MoleculeNet
```

Comment:
I think we should move this file to `.github/PULL_REQUEST_TEMPLATE` and create a pull request template.  https://help.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository

And then, it is better to do same things about `ISSUE_TEMPLATE.md` (move it to `.github/ISSUE_TEMPLATE`)

Reply:
Sounds good! I've created that directory structure. It may be useful to have a generic PR template too, especially for new contributors.

Reply:
+1 for a generic PR template!